Mono.Cecil/Import.cs

```diff
@@ -769,6 +769,42 @@ public static bool TryGetAssemblyNameReference (this ModuleDefinition module, As
 			assembly_reference = null;
 			return false;
 		}
+
+		private static bool Equals (byte [] a, byte [] b)
+		{
+			if (Object.ReferenceEquals (a, b))
+				return true;
+			if (a.Length != b.Length)
```

Comment:
I should check for null here (PublicKeyToken is never null today, but that may change or this helper could be reused)